plugins/unicodelib/unicodelib.cpp

```diff
@@ -719,6 +720,42 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+void excludeLastWord(RuleBasedBreakIterator& bi, UnicodeString & source)
```

Comment:
style: Not keen on a parameter that is modified being called source (see other PRs for comments)